db/migrate/20180303104526_add_user_id_to_expense.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToExpense < ActiveRecord::Migration[5.1]
+  def change
+    add_column :expenses, :user_id, :integer
```

Comment:
For future reference, Instead of manually adding the `user_id` field or any foreign key to a table with migrations using `add_column` rather make use of `add_reference`.

In this case your migration would look like

```ruby
  class AddUserRefToExpense < ActiveRecord::Migration[5.1]
    def change
      add_reference :expenses, :user, foreign_key: true
    end
  end
```

Doing it this way, it'll also create the proper index on your database table to allow for a more efficient query. Read [this section](http://edgeguides.rubyonrails.org/active_record_migrations.html#creating-a-standalone-migration) of the rails guide on how to generate standalone migrations, especially the part towards the end that talks about adding references (or belongs_to) associations